packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -194,7 +194,7 @@ export declare class VideoPlayer extends SharedObject<VideoPlayerEvents> {
    * Initializes a new video player instance with the given source.
    * @hidden
    */
-  constructor(source: VideoSource);
+  constructor(source: VideoSource, useSynchronousReplace?: boolean);
```

Comment:
Can we add more information about the `useSynchronousReplace` parameter? Right now, it's unclear whether it should be used.